code/browser.py

```diff
@@ -25,3 +26,12 @@ def address():
         # Prioritize last one if multiple are valid, return empty string if none is valid.
         tokens = (url[1: -1] if not is_url(url) else url for url in reversed(actions.win.title().split(" ")))
         return next((url for url in tokens if is_url(url)), "")
+
+
+@mod.action_class
```

Comment:
Added to the `browser` tag rather than the `tabs` tag since even though it's an action related to tabs, it's only relevant for browsers. Happy to discuss merits of each option.

Reply:
Will move to the `tabs` interface now that I've thought of some use cases beyond browsers